locales/ar-arb/messages.po

```diff
@@ -512,7 +516,7 @@ msgid "Fail to export item"
 msgstr "فشل تصدير البند"
 
 msgid "Failed to publish %1$s in %2$s"
-msgstr "فشل نشر٪ 1 $ s في٪ 2 $ s"
+msgstr "فشل نشر %1$s في %2$s"
```

Comment:
variable separated?

Reply:
Same here: 
![image](https://user-images.githubusercontent.com/1084316/152959247-714ae56e-1d7b-47af-a959-90eed5687c74.png)

Reply:
This is the branch on my local machine
![image](https://user-images.githubusercontent.com/11692751/152963331-cd73dd67-c733-4f77-9ec0-e15fca817eef.png)

Reply:
But on the messages_po.js
![image](https://user-images.githubusercontent.com/11692751/152963816-c6c934df-78ad-4bb6-8fe1-5336f5be2f48.png)

Reply:
Yes, this is an issue with text editors only. These mixed chars are not correctly rendered.